codegen/build_dbc.py

```diff
@@ -29,50 +29,8 @@
 }
 
 # pylint: disable=W0511
-# TODO: Determine a way of encoding this in the ASCIIPB
-SIGNED_MESSAGES = [
-    'DRIVE_OUTPUT',
-    'CRUISE_TARGET',
-    'BATTERY_AGGREGATE_VC',
-    'MOTOR_VELOCITY'
-]
-
-# pylint: disable=W0511
-# TODO: Determine a way of encoding this in the ASCIIPB
-ACKABLE_MESSAGES = {
-    0: [
-        'CHAOS',
-        'LIGHTS_FRONT',
-        'PLUTUS_SLAVE',
-        'DRIVER_CONTROLS_PEDAL'
-    ],
-    1: [
-        'DRIVER_CONTROLS_PEDAL'
-    ],
-    2: [
-        'PLUTUS'
-    ],
-    3: [
-        'PLUTUS_SLAVE'
-    ],
-    4: [
-        'MOTOR_CONTROLLER'
-    ],
-    5: [
-        'SOLAR_MASTER_REAR'
-    ],
-    6: [
-        'SOLAR_MASTER_FRONT'
-    ],
-    7: [
-        'CHAOS'
-    ],
-    8: [
-        'PLUTUS',
-        'MOTOR_CONTROLLER',
-        'DRIVER_CONTROLS_PEDAL'
-    ],
-}
+SIGNED_MESSAGES = []
```

Comment:
Why do we need signed messages anyways? does the asciipb literally not allow signed integers?